meteor/client/lib/SplitPreviewBox.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames'
+import React from 'react'
+import { RundownUtils } from './rundown'
+import { SplitRole, SplitSubItem } from './ui/splitPreview'
+
+export const RenderSplitPreview = React.memo(function RenderSplitPreview({
```

Comment:
None, since the props are generated off a reactive Collection source, which will produce new objects whenever results of a Cursor change. That being said, I agree that the fact that the array is treated as immutable should be made explicit in the typings. I've added a ReadonlyArray & Readonly around the SplitSubItem, which should enforce compilation-time immutability. That should be enough for us.